docs/articles/scaleout.md

```diff
@@ -122,22 +179,23 @@ From zot release v2.1.9 and newer, the `sessionDriver` configuration can also be
     "tls": {
       "cacert": "test/data/ca.crt"
     }  
+  },
+  "extensions": {
+    "ui": {
+      "enable": true
+    }
   }
 }
 ```
 
 </details>
 
-### HAProxy configuration
-
-The HAProxy load balancer uses a simple round-robin balancing scheme. If zli and the zot UI are used to interact with zot, the proxy is also required to deliver a cookie to the requestor to maintain a sticky session connection to the assigned instance.
-
-> :point_right: For zot releases v2.1.9 and newer, zot can be optionally configured to save session information to an external Redis-compatible storage which can be shared by all zot replicas. If this configuration is enabled, the proxy need not configure a sticky cookie and the proxy can be optionally removed from the deployment. Note that this configuration is suitable for use when all the zot replicas share a common data storage such as S3 and a common remote metadata store such as DynamoDB or Redis.
+For older releases of zot that do not support external session storage, a load balancer can set sticky sessions.
```

Comment:
I included this section because I felt that the external session storage is still a pretty new feature so it would help users who land here to know about the sticky session cookie.

Ideally, since the docs are versioned and the user can change between them, we could consider removing it.

What do you think?